akka-stream-testkit/src/test/scala/akka/stream/testkit/ScriptedTest.scala

```diff
@@ -121,14 +123,14 @@ trait ScriptedTest extends Matchers {
     def mayRequestMore: Boolean = remainingDemand > 0
```

Comment:
Also, this does much more than a user needs. This does fuzzing of the internal protocol by trying out different timings, but users are expected to use the higher level DSL and therefore rarely need this fuzzing.